operations/commit_queue.go

```diff
@@ -319,6 +319,9 @@ func (p *mergeParams) uploadMergePatch(conf *ClientSettings, ac *legacyClient) e
 		}
 		return errors.Wrap(err, "can't get project ref")
 	}
+	if !ref.CommitQueue.Enabled {
```

Comment:
maybe we should also check the project is using the CLI queue?

Reply:
oh very good point